### PR TITLE
Hybridize d2l-rubric to work with polymer 1 and 2

### PR DESCRIPTION
**Pull request by alogan519**

### CODE REVIEW

File: bower-1.x.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
```
Comment (dbatiste) [inline bower-1.x.json]: This file should not be checked in.  We should have `bower-1.x.json` in `.gitignore`.

File: package.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
     "test": "test"
   },
   "scripts": {
-    "postinstall": "bower install",
+    "postinstall": "polymer install --variants",
     "serve": "polymer analyze > analysis.json && polymer serve",
     "test": "npm run test:lint && npm run test:wct",
     "test:lint": "npm run test:lint:wc && npm run test:lint:html",
```
Comment (dbatiste) [inline package.json]: Do you know the reason for the `cross-env LAUNCHPAD_BROWSERS=chrome polymer test -p --skip-plugin sauce`?  Seems this is getting copied around, but I don't _think_ we need/want this.

Comment (dbatiste) [reply package.json]: Also, further below, we can remove the `bower` dev dependency, since `polymer install` takes care of that for us.

Comment (CodeBaboon) [reply package.json]: on Windows machines the tests will pause for minutes (like, 5+) before running while searching for browsers. `LAUNCHPAD_BROWSERS=chrome` hijacks that by just directing it to Chrome. The `cross-env` is necessary for setting the env-var in Windows.

Comment (dbatiste) [reply package.json]: Thanks Dave.  This makes me sad.   :)

**Comment by alogan519** [comment]
Oh sorry guys checked this in for Stacey to help debug some things, will make sure to address concerns before going into master.

File: package.json (hunk 2 above)
Comment (CodeBaboon) [reply package.json]: The saddest part is that it's been a [known issue](https://github.com/Polymer/web-component-tester/issues/169) in the WCT local browser detection for close to 3 years and has yet to be resolved.

Comment (dbatiste) [reply package.json]: Yeah.  Us Mac users just keep sailing along not realizing.  Thanks for clarifying.  :)

**Pull request merged**